translations/es/pack/SoD.json

```diff
@@ -0,0 +1,61 @@
+[
+    {
+        "code": "10001",
+        "name": "Doran Martell",
+        "text": "<b>Action:</b> Kneel your faction card to choose a participating character with fewer than 2 challenge icons and remove it from the challenge.\n<b>Reaction:</b> After you lose a challenge, choose a character. Until the end of the phase, that character gains or loses a challenge icon of your choice.",
+        "traits": "Lord."
+    },
+    {
+        "code": "10008",
+        "flavor": "Her soft, pale hands were as deadly as Obara's callused ones, if not more so.",
+        "name": "Tyene Sand",
+        "text": "Stealth.\n<b>Reaction:</b> After you lose a challenge as the defending player, kneel a <i>Sand Snake</i> character to put Tyene Sand into play from your hand.",
+        "traits": "Bastard. Sand Snake."
+    },
+    {
+        "code": "10012",
+        "flavor": "\"Give your bride a kiss for me.\"",
+        "name": "Ser Cletus Yronwood",
+        "text": "No attachments except <i>Weapon</i>.\nIncrease the initiative value on your revealed plot card by 1 for each plot card in your used pile.",
+        "traits": "House Yronwood. Knight."
+    },
+    {
+        "code": "10017",
+        "name": "Dorne",
+        "text": "<b>Reaction:</b> After you lose a challenge, look at the top 2 cards of your deck. Add 1 to your hand, and place the other on the bottom of your deck.\n+1 Reserve.",
+        "traits": "Dorne."
+    },
+    {
+        "code": "10020",
+        "flavor": "\"You mistake patience for forbearance. I have worked at the downfall of Tywin Lannister since the day they told me of Elia and her children.\" <cite>Doran Martell</cite>",
+        "name": "Patience",
+        "text": "Character you control only.\n<b>Challenges Action:</b> Return attached character to its owner's hand.",
+        "traits": "Skill."
+    },
+    {
+        "code": "10039",
+        "flavor": null,
```

Comment:
This fails validation on the translation file but seems to be fine in the English source. Looking at the schema, `flavor` is allowed to be null or an empty string but the translation file schema requires it to be an empty string or missing.

Dropping this line should get rid of the validation error, the schema inconsistency should probably be addressed separately.